docs/dev-guide/src/config/providing.md

```diff
@@ -10,3 +10,31 @@ Flags can be set in one of four ways, in increasing order of priority:
 3. Provided individually as environment variables with the prefix `PRUSTI_` (for example, `PRUSTI_ASSERT_TIMEOUT` for the [`ASSERT_TIMEOUT`](flags.md#assert_timeout) flag).
 
 4. Provided individually as command-line arguments to Prusti with the prefix `-P` (for example, `-Pprint_desugared_specs` for the [`PRINT_DESUGARED_SPECS`](flags.md#print_desugared_specs) flag).
+
+## Multi-crate Cargo Prusti Projects
+
+Setting flags becomes slightly more complicated when Prusti is run on multiple crates as `cargo prusti` (e.g. which `Prusti.toml` file will be used). The three possible approaches to providing flags all behave differently.
+
+### Environment Variables
+
+The environment persists throughout compilation, therefore any flags set through the `DEFAULT_PRUSTI_` or `PRUSTI_` will apply to both the root crate and all dependencies. For example setting `CHECK_OVERFLOWS=false` will disable overflow checks in all dependencies, even if they try to override this with a `Prusti.toml` file.
```

Comment:
> `CHECK_OVERFLOWS=false`

Should be `PRUSTI_CHECK_OVERFLOWS=false`?

> setting `CHECK_OVERFLOWS=false` will disable overflow checks in all dependencies, even if they try to override this with a `Prusti.toml` file.

In a single crate `Prusti.toml` doesn't override `PRUSTI_*` environment variables. So, the quoted sentence might be a bit confusing.